drakrun/drakrun/main.py

```diff
@@ -326,8 +326,8 @@ def analysis_uid(self):
 
         return self.current_task.uid
 
-    def build_drakvuf_cmdline(self, timeout, cwd, full_cmd, dump_dir, ipt_dir):
-        hooks_list = os.path.join(ETC_DIR, "hooks.txt")
+    def build_drakvuf_cmdline(self, timeout, cwd, full_cmd, dump_dir, ipt_dir, workdir):
```

Comment:
If we add `workdir` param here, then surely we need to add an argument to where we call this function from.

Reply:
we do, on line 471

Reply:
Ah, I'm blind.